### PR TITLE
Fix to create_trip mutation

### PR DESCRIPTION
**Pull request by stephenfabian**
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please describe):

## Problem/feature
_What problem are you trying to solve?_
Fix to create trip mutation, accounts for changing category arg from integer to string

## Checklist
- [ ] Code compiles correctly
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by Hmonroe2 — APPROVED** [review]

File: app/graphql/mutations/trip_create.rb [hunk 1]
```diff
@@ -10,15 +10,19 @@ class TripCreate < BaseMutation
     field :trip, Types::TripType, null: false
 
     def resolve(name:, category:, traveler:)
-      # trip = ::Trip.new(name: name, category: category)
       trip = Trip.new(name: name, category: category, traveler: traveler, image: image_selector(category))
+      require 'pry'; binding.pry
```
Comment (juliet-e) [inline app/graphql/mutations/trip_create.rb]: Remember to remove your prys before you push to github!

File: app/graphql/mutations/trip_create.rb [hunk 2]
```diff
@@ -10,15 +10,18 @@ class TripCreate < BaseMutation
...
-      # trip = ::Trip.new(name: name, category: category)
       trip = Trip.new(name: name, category: category, traveler: traveler, image: image_selector(category))
       raise GraphQL::ExecutionError.new "Error creating trip", extensions: trip.errors.to_hash unless trip.save
       { trip: trip }
     end
 
     def image_selector(category)
-      "https://www.sandals.com/blog/content/images/2020/03/Sandals-Barbados-Beach-Palm-Trees-Beach.jpg" if category == 2
-      "https://www.lonestar.edu/images/internationalTravel.jpg" if category == 4
+      return "https://www.sandals.com/blog/content/images/2020/03/Sandals-Barbados-Beach-Palm-Trees-Beach.jpg" if category == "beach"
```
Comment (juliet-e) [inline app/graphql/mutations/trip_create.rb:19]: What if you created a hash where the key is the category and the value is the image affiliated with the image? That way, you can just have one return statement
`return image_for_category[category]` if image_for_category is your hash of images? You would then just need one fallback image if the given category doesn't exist in the hash

File: spec/graphql/mutations/trip_items/trip_item_delete_spec.rb [hunk 3]
```diff
@@ -3,6 +3,8 @@
 RSpec.describe 'TripItemDelete', type: :request do
   describe '.resolve' do
     before :each do
+      #let let!(create variable before its called)
+      #black box vs clear box testing(industry - black box)
```
Comment (juliet-e) [inline spec/graphql/mutations/trip_items/trip_item_delete_spec.rb:7]: Just a reminder that these comments are here! Would be a great opportunity to experiment with `let`

**Review by juliet-e — COMMENTED** [review]
Looks good! I added a few comments but only one that deals with a bit of refactoring. Nice work 👍

**Pull request merged**